pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev.go

```diff
@@ -50,7 +50,7 @@ func CreateHostDevices(vmi *v1.VirtualMachineInstance) ([]api.HostDevice, error)
 	if len(SRIOVInterfaces) == 0 {
 		return []api.HostDevice{}, nil
 	}
-	netStatusPath := path.Join(downwardapi.MountPath, deviceinfo.NetworkPCIMapVolumePath)
+	netStatusPath := path.Join(downwardapi.MountPath, downwardapi.NetworkInfoVolumePath)
 	pciAddressPoolWithNetworkStatus, err := newPCIAddressPoolWithNetworkStatusFromFile(netStatusPath)
```

Comment:
That's the closest place to the real place I wanted to comment - 
`newPCIAddressPoolWithNetworkStatusFromFile` should be changed - s/networkPCIMapBytes/networkDeviceInfoBytes

Reply:
Sorry, I thought you mean for `NewPCIAddressPoolWithNetworkStatus` argument name.

Done.